source/Main.cpp

```diff
@@ -13,6 +13,7 @@
 #include <Render.h>
 #include <SceneManager.h>
 #include <SoundManager.h>
+#include <clocale>
```

Comment:
bez tego nie działa u ciebie? mogłeś dać alfabetycznie ale to już poprawie kiedyś :P